packages/react-components/src/components/TagInput/TagInput.tsx

```diff
@@ -99,29 +100,38 @@ export const TagInput: React.FC<ITagInputProps> = ({
   };
 
   return (
-    <div className={mergedClassNames}>
-      {tags?.map((tag, index) => (
-        <Tag
-          index={index}
-          key={`${index}${tag}`}
-          update={updateTag}
-          remove={removeTag}
-          inputRef={inputRef}
-          validator={validator}
-          size={size}
-        >
-          {tag}
-        </Tag>
-      ))}
-      <input
-        ref={inputRef}
-        className={inputClassNames}
-        placeholder={placeholder}
-        value={inputValue}
-        onChange={onInputChange}
-        onKeyDown={onInputKeyDown}
-        onPaste={onPaste}
-      />
-    </div>
+    <>
+      <div className={mergedClassNames}>
+        {tags?.map((tag, index) => (
+          <EditableTag
+            index={index}
+            key={`${index}${tag}`}
+            update={updateTag}
+            remove={removeTag}
+            inputRef={inputRef}
+            validator={validator}
+            size={size}
+          >
+            {tag}
+          </EditableTag>
+        ))}
+        <input
+          ref={inputRef}
+          className={inputClassNames}
+          placeholder={placeholder}
+          value={inputValue}
+          onChange={onInputChange}
+          onKeyDown={onInputKeyDown}
+          onPaste={onPaste}
+        />
+      </div>
+      {error && (
+        <span className="lc-field-error">
+          <Text as="span" size="sm">
+            {error}
+          </Text>
+        </span>
```

Comment:
Why not just `<Text className="lc-field-error" as="span" size="sm">{error}</Text>` ? :)

Reply:
@przemuh Copy/pasted from TextField component 😄 
But after applying className directly on Text component I found strange case -> the size of Text is changing - `lc-field-error ` sets font-size (13px) and Text component is setting its own font-size (14px) which has higher priority than the parent class. Checked on [Figma](https://www.figma.com/file/2pFu80PXO5A2tfyrAGnx91/Product-Components?node-id=4046%3A33393) and it should have font-size 14px and typography tokens as well, so I fixed it in TextField component as well.

But in the end, I found out that we have some duplicates in the codebase(we already have FieldError and FieldDescription components which have typography tokens) so I fixed it a little bit 😄